### DISCUSSION TITLE
`Repair Tree` Button Could Do More

### DISCUSSION THREAD

**Discussion by carlosmbe** [specify/specify7#5822] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
In the User Tools section the `Repair Tree` Button isn't really useful in it's current state. After talking to Jason the functionality can be described as " 'repairing' the tree just constitutes renumbering it and throwing an error if it really broken. "

However, this also happens every time the user opens the tree. 

**Describe the solution you'd like**
It'd be great if `Repair Tree` was able to deal with more complex issues. 

Perhaps give it a back up and restore tree feature. With customisable attributes for when you restore the tree.

Or just a revert changes feature. That reverts a tree to a _Pre Broken State_.

These are just ideas. But I believe in the Specify Dev Team to find a nice solution.
<img width="898" alt="Screenshot 2023-06-13 at 11 48 15 AM" src="https://github.com/specify/specify7/assets/53784701/8175fd3b-917c-45b5-954a-b074950c9847">

**Comment by grantfitzsimmons** [comment]
It doesn't do _anything_ right now! 


See https://github.com/specify/specify7/issues/2776
